File: edit/edit.js
var stdin= process.openStdin();
stdin.addListener("data", function(d)
{
var x= d.toString().trim().split("\n");
var res= minimum_distance(x[0],x[1]);
console.log(res);
});

function minimum_distance(str1,str2)
    {   
    let minimum_value=-1;
    //Write your code here without removing the existing code
	//the variables string 'str1' and 'str2' contains the input values in the string format
	//modified the variable 'minimum_value' contain the output of the program.
    return minimum_value;
    };
    

File: edit/edit.c
#include <stdio.h>
//#include<stdio.h>
#include <stdlib.h>

// #include <stdio.h>
#include <string.h>
// Starter Code
int  minimum_distance(char str1[100000],char str2[100000])
{
    int  minimum_value=-1;
    //Write your code here without removing the existing code
	//the variable 'str1' and 'str2' contains array of character.
	//modified the variable 'minimum_value ' contain the output of the program.
    return minimum_value ;
}


//Backend Code
int main()
{
    char str1[100000],str2[100000];
    scanf("%[^\n]%*c",str1);
    scanf("%[^\n]%*c",str2);
    int ans= minimum_distance(str1,str2);
    printf("%d",ans);
 
    return 0;
}